### ISSUE TITLE
Photons travel slower than beam. Design?

### ISSUE THREAD

**Issue by elisemorgan** [phetsims/color-vision#104]
Test device: Mac, Personal

Operating System: 10.10.5

Browser: Safari 9.0.3 (10601.4.4)

Problem description: I noticed that if I have the light in beam mode, turn the light off, then immediately switch to photon mode there are still photons traveling towards the eye. This seems misleading, especially since the person has stopped seeing color in the beam mode and then sees it again with the photons. I'm thinking that maybe this is a design decision to have the photons move slower?

Steps to reproduce: Turn on light in beam mode. Turn off light. Immediately switch to photon mode.

Severity:

Screenshots: https://drive.google.com/file/d/0B_TAL-jnJOoKRTNqYkNmdTl2d3c/view?usp=sharing

Troubleshooting information (do not edit):
Name: ‪Color Vision‬
URL: http://www.colorado.edu/physics/phet/dev/html/color-vision/1.2.0-dev.1/color-vision_en.html
Version: 1.2.0-dev.1 2016-02-12 00:01:13 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/601.4.4 (KHTML, like Gecko) Version/9.0.3 Safari/601.4.4
Language: en-us
Window: 1238x664
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (2.1 INTEL-10.6.33)
GLSL: WebGL GLSL ES 1.0 (1.20)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 16)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"7d27130a","branch":"master"},"axon":{"sha":"60213ff7","branch":"master"},"babel":{"sha":"09b19d7a","branch":"master"},"brand":{"sha":"ac9ccaa7","branch":"master"},"chipper":{"sha":"a0be095e","branch":"master"},"color-vision":{"sha":"c86281c1","branch":"master"},"dot":{"sha":"da78031b","branch":"master"},"joist":{"sha":"c61635d6","branch":"master"},"kite":{"sha":"10ca8bf2","branch":"master"},"phet-core":{"sha":"c48bf320","branch":"master"},"phetcommon":{"sha":"35c2abc1","branch":"master"},"scenery":{"sha":"6bd54e23","branch":"master"},"scenery-phet":{"sha":"f995e3ee","branch":"master"},"sherpa":{"sha":"bb68bbbc","branch":"master"},"sun":{"sha":"2f236352","branch":"master"},"tandem":{"sha":"d0be78dd","branch":"master"}}

**Comment by ariel-phet** [comment]
@elisemorgan - definitely not a design decision. And agreed, misleading.

@pixelzoom please comment if this is an easy fix or not.

**Comment by pixelzoom** [comment]
Off the top of my head, no idea if this is an easy fix. I would need 30-45 minutes to investigate.  It's certainly not a new problem, and didn't we decide to fix only i18n issues or new problems?

**Comment by pixelzoom** [comment]
@ariel-phet back to you. Let me know if you want me to investigate. Otherwise please label as "priority:5-deferred" and we'll handle it in a future release, along with the other open issue for this sim.

**Comment by pixelzoom** [comment]
@ariel-phet back to you. Let me know if you want me to investigate (it's time I won't spend on function-builder). Otherwise please label as "priority:5-deferred" and we'll handle it in a future release, along with the other open issues for this sim.

**Comment by ariel-phet** [comment]
@pixelzoom correct, we decided to fix only i18n issues (and any truly low hanging fruit).

**Comment by pixelzoom** [comment]
Related to RC test https://github.com/phetsims/tasks/issues/511.

**Comment by phet-steele** [comment]
As a consequence, if the sim is paused while the light is turned off, switching to photon mode and turning on the light will not have photons appear. There must be time steps for photons to be created.

**Comment by pixelzoom** [comment]
Certainly not by design, and not an issue with the Java version.  Most certainly a bug in the HTML5 port, and probably easy to fix.

**Comment by pixelzoom** [comment]
The original problem (reported in https://github.com/phetsims/color-vision/issues/104#issue-133787667) is also not by design, it's a bug in the port.
